### DISCUSSION TITLE
Data in Postman an din Applications is different from Web Browser

### DISCUSSION THREAD

**Discussion by knee11** [osm-search/Nominatim#2787] [Geocoding around the world]
Hi All,

We are trying to include our created Nominatim API in our Project System which needs to identify the specific address of the lat and long data. But upon checking, the result in Nominatim reverse has different "country" data from our Application or Postman than on a Web Browser, please see the below image:

 This is the data in Postman:
 
![image](https://user-images.githubusercontent.com/110162696/182530327-4c15a7b2-80be-4683-a687-156508899c54.png)

And this is on a Web Browser:

![image](https://user-images.githubusercontent.com/110162696/182530428-854e5924-8c05-4737-a4a6-a5d8cd0a3f24.png)

We used the same URL but they have different outputs on country data.

Thank you!

**Comment by lonvia** [comment]
The problem here is that [Mindanao](https://www.openstreetmap.org/relation/11144440) is maaped with the fraction of an admin level. That confuses Nominatim.
